### PR TITLE
Integrate checker with c2cwsgiutils

### PR DESCRIPTION
**Pull request by pvalsecc**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -95,8 +95,8 @@ script:
   find setup.py testgeomapfish/*.py testgeomapfish/views -name \*.py | xargs travis/quote
 - travis/run-on.sh /tmp/travis/testgeomapfish/ docker-compose up -d
 - travis/waitwsgi http://localhost:8080/
-- travis/test-new-project http://localhost:8080/check_collector
-- travis/test-new-project http://localhost:8080/check_collector?type=all
+- travis/test-new-project http://localhost:8080/c2c/health_check
+- travis/test-new-project http://localhost:8080/c2c/health_check?max_level=100
```
Comment (sbrunner) [inline .travis.yml]: From where does this c2c comes from?

File: doc/integrator/checker.rst [hunk 2]
```diff
@@ -3,41 +3,40 @@
...
 
-Those services (especially the collector) are meant to be used by a
+Those (especially the collector) are meant to be used by a
 monitoring system like Nagios to check that the application is alive.
 
-The return code are:
+They integrate into c2cwsgiutils's health check service. The return code are:
 
 * ``200``-``299`` => OK
-* ``400``-``499`` => Warning
```
Comment (sbrunner) [inline doc/integrator/checker.rst]: Why removing warning?

Comment (pvalsecc) [reply doc/integrator/checker.rst]: There is no such thing as warning in health_check

File: .travis.yml (hunk 1 above)
Comment (pvalsecc) [reply .travis.yml]: from c2c.base_path in the production.ini

File: doc/integrator/checker.rst (hunk 2 above)
Comment (sbrunner) [reply doc/integrator/checker.rst]: => 500 or 200

File: .travis.yml (hunk 1 above)
Comment (sbrunner) [reply .travis.yml]: ok

File: c2cgeoportal/scaffolds/nondockercreate/vars_nondocker.yaml_tmpl [hunk 3]
```diff
@@ -29,9 +29,7 @@ vars:
         phantomjs:
             routes: []
     check_collector:
-        hosts:
-        - display: {{package}}
-          url: '{web_protocol}://{host}/{instanceid}/wsgi'
+        hosts: []
```
Comment (sbrunner) [inline c2cgeoportal/scaffolds/nondockercreate/vars_nondocker.yaml_tmpl]: You override with the same thing, we can just remove it ...

File: travis/vars.yaml [hunk 4]
```diff
@@ -14,16 +14,13 @@ vars:
...
+            files: [ngeo]
+        routes:
+            disable: [printproxy_capabilities]
 
 update_paths:
 - layers
-- checker.defaults
+- checker.lang
+- checker.routes
 - checker.all
```
Comment (sbrunner) [inline travis/vars.yaml]: No more used :-)

File: travis/vars-nondocker.yaml [hunk 5]
```diff
@@ -2,16 +2,13 @@ extends: vars_nondocker.yaml
 
 vars:
     checker:
-        defaults:
-            lang_files: [ngeo]
-            routes_disable: [printproxy_capabilities]
-        all:
-            routes_disable: [printproxy_capabilities]
-    check_collector:
-        disabled: [checker_pdf3, checker_fts]
```
Comment (sbrunner) [inline travis/vars-nondocker.yaml]: We should disable the checker_pdf3 check... => level 3 in this file?

**Comment by sbrunner** [comment]
> I still need the runtime lib, no?

no, we just don't install GDAL :-)

File: travis/vars-nondocker.yaml (hunk 5 above)
Comment (pvalsecc) [reply travis/vars-nondocker.yaml]: why do you want to disable it in non-docker mode?

**Pull request merged**